lib/content-types.js

```diff
@@ -36,40 +36,51 @@ const raw = loader;
  * Merge content type object with input plugin data
  *
  * @param {array} content content types array
+ * @param {object} config - configuration object
  *
  * @returns {array} content types array that has been merged with input plugins and user settings
  */
-const merged = (content) => {
+const merged = (content, config) => {
   let mergedTypes = _.cloneDeep(content);
 
   if (content) {
     return merge(mergedTypes);
   }
 
-  return raw().then(types => {
+  return raw(config).then(types => {
     mergedTypes = _.cloneDeep(types);
 
-    return merge(mergedTypes);
+    return merge(mergedTypes, config);
   });
 };
 
-const onlyCT = (type, config, loadedTypes) => {
-  const userConfig = config || {};
+/**
+ * Returns only one content type, merged with input plugins and values
+ *
+ * @param  {string} type - id of a content type
+ * @param  {object} values - contains values for inputs
+ * @param  {array} loadedTypes - array of merged content type objects
+ * @param  {object} config - configuration object
+ *
+ * @returns {promise} resolves with promise from `only` function
+ */
+const onlyCT = (type, values, loadedTypes, config) => {
+  const userConfig = values || {};
```

Comment:
I would expect `userConfig` to be a version of `config` not `values`; can this be something like `overrides`